advanced_functionality/causal-inference/causal-inference-container.ipynb

```diff
@@ -0,0 +1,588 @@
+{
```

Comment:
<p>Grammar nit: be consistent with spelling "discretiser/discretisation" versus "discretizer/discretization" in markdown/text. (Understood that the method can only be "discretiser" in code)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3130/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>